src/renderer/wg_engine/tvgWgPipelines.cpp

```diff
@@ -317,7 +388,7 @@ void WgPipelineComposeBlend::initialize(WGPUDevice device)
     };
 
     // sheder source and labels
```

Comment:
typo (in other places as well): sheder -> shader

Reply:
Fixed